src/overlays/actors/ovl_Obj_Jgame_Light/z_obj_jgame_light.h

```diff
@@ -9,8 +9,19 @@ struct ObjJgameLight;
 
 typedef struct ObjJgameLight {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x78];
-} ObjJgameLight; // size = 0x1BC
+    /* 0x144 */ ColliderCylinder collider;
+    /* 0x190 */ UNK_TYPE1 pad_190[4];
+    /* 0x194 */ LightNode* lightNode;
+    /* 0x198 */ LightInfo lightInfo;
+    /* 0x1A8 */ f32 unk_1A8;
+    /* 0x1AC */ s16 unk_1AC;
+    /* 0x1AE */ s16 unk_1AE;
+    /* 0x1B0 */ UNK_TYPE1 pad_1B0[2];
+    /* 0x1B2 */ s16 unk_1B2;
+    /* 0x1B4 */ s16 unk_1B4;
+    /* 0x1B6 */ s16 unk_1B6;
+    /* 0x1B8 */ u8 unk_1B8;
+} ObjJgameLight; /* size = 0x1BC */
```

Comment:
```suggestion
} ObjJgameLight; // size = 0x1BC
```
These should all use line comments.